### PR TITLE
Add OOYE bridge

### LINKED ISSUE #2026: Add "Out of your Element" Discord bridge
I came across this today, a Matrix<->Discord bridge: https://gitdab.com/cadence/out-of-your-element.

We should add it to https://matrix.org/ecosystem/bridges/discord/.

### PR DESCRIPTION
**Pull request by thibaultamartin**
Fixes #2026 

@cloudrac3r is everything here accurate?

### CODE REVIEW

File: content/ecosystem/bridges/discord/bridges.toml [hunk 1]
```diff
@@ -89,3 +89,30 @@ editing = true
 reactions = true
 presence = false
 typing_notifications = true
+
+[[bridges]]
+name = "Out Of Your Element"
+maintainer = "Cadence"
+summary = "Modern Matrix-to-Discord appservice bridge."
+maturity = "Beta"
+language = "Javscript"
```
Comment (cloudrac3r) [inline content/ecosystem/bridges/discord/bridges.toml:98]: \* JavaScript

File: content/ecosystem/bridges/discord/bridges.toml [hunk 2]
```diff
@@ -89,3 +89,30 @@ editing = true
...
+formatted_text = true
+message_media = true
+replies = true
+mentions = true
+threads = true
+redactions = true
+editing = true
+reactions = true
+presence = true
+typing_notifications = false
```
Comment (cloudrac3r) [inline content/ecosystem/bridges/discord/bridges.toml:118]: Discord users typing will be shown on Matrix-side :heavy_check_mark: , while Matrix users typing will not be shown on Discord-side :x:. This is a Discord limitation.

It's your call on whether you want to mark this feature as :heavy_check_mark: or :x: overall.

File: content/ecosystem/bridges/discord/bridges.toml [hunk 3]
```diff
@@ -89,3 +89,30 @@ editing = true
...
+name = "Out Of Your Element"
+maintainer = "Cadence"
+summary = "Modern Matrix-to-Discord appservice bridge."
+maturity = "Beta"
+language = "Javscript"
+license = "AGPL-3.0-or-later"
+docs = "https://gitdab.com/cadence/out-of-your-element/src/branch/main/docs"
+repo = "https://gitdab.com/cadence/out-of-your-element"
+room = "#out-of-your-element:cadence.moe"
+featured = false
```
Comment (cloudrac3r) [inline content/ecosystem/bridges/discord/bridges.toml:103]: What does `featured` mean?

**Review by cloudrac3r — COMMENTED** [review]
Looks mostly accurate, thank you! I've asked some questions in the review comments.

File: content/ecosystem/bridges/discord/bridges.toml [hunk 4]
```diff
@@ -89,3 +89,30 @@ editing = true
 reactions = true
 presence = false
 typing_notifications = true
+
+[[bridges]]
+name = "Out Of Your Element"
+maintainer = "Cadence"
+summary = "Modern Matrix-to-Discord appservice bridge."
```
Comment (cloudrac3r) [inline content/ecosystem/bridges/discord/bridges.toml:96]: A new suggested summary, if you'll allow me to market myself a bit harder ;)

> An efficient and modern bridge. It supports the latest Discord features like replies and threads, and the latest Matrix features like custom emotes and intentional mentions.

File: content/ecosystem/bridges/discord/bridges.toml [hunk 5]
```diff
@@ -89,3 +89,30 @@ editing = true
...
+channels = true
+formatted_text = true
+message_media = true
+replies = true
+mentions = true
+threads = true
+redactions = true
+editing = true
+reactions = true
+presence = true
```
Comment (cloudrac3r) [inline content/ecosystem/bridges/discord/bridges.toml:117]: Oh, it doesn't have presence support. I probably won't add presence support until mainline Element has it.

File: content/ecosystem/bridges/discord/bridges.toml (hunk 3 above)
Comment (thibaultamartin) [reply content/ecosystem/bridges/discord/bridges.toml:103]: Featured is a flag we intended to use to promote the projects more heavily, to draw attention to the most sustainable and polished projects. It's only used to add a little "Recommended" chip for now, and is left to the discretion of the m.org crew before we have a more formal process for it :)

You can follow or contribute to https://github.com/matrix-org/matrix.org/issues/1584 if you want!

Comment (cloudrac3r) [reply content/ecosystem/bridges/discord/bridges.toml:103]: Sweet as. Thank you Thibault!

**Pull request merged**